src/app/public-view/home/home-instagram-gallery/home-instagram-gallery.component.ts

```diff
@@ -0,0 +1,45 @@
+import {Component, OnInit} from '@angular/core';
+import {InstagramSearchService} from '../services/instagram-search.service';
+import {Gallery} from 'angular-gallery';
+
+@Component({
+  selector: 'app-home-instagram-gallery',
+  templateUrl: './home-instagram-gallery.component.html',
+  styleUrls: ['./home-instagram-gallery.component.scss']
+})
+export class HomeInstagramGalleryComponent implements OnInit {
+
+  public posts: Array<{ path: string }> = [];
```

Comment:
`public posts: Array<InstagramPost>` ?

Reply:
Musí to byť v takomto formáte kvôli npm knižnici pre galériu. InstagramPost je objekt ktorý zaobaľuje response z instagram API.